spacy_streamlit/visualizer.py

```diff
@@ -312,13 +312,18 @@ def visualize_spans(
             st.warning(
                 "When the parameter 'manual' is set to True, the parameter 'doc' must be of type 'Dict', not 'spacy.tokens.Doc'."
             )
-
-    html = displacy.render(
-        doc,
-        style="span",
-        options=displacy_options,
-        manual=manual,
-    )
+    try:
+        html = displacy.render(
+            doc,
+            style="span",
+            options=displacy_options,
+            manual=manual,
+        )
+    except ValueError as e:
+        if str(e) == "[E087] Unknown displaCy style: span.":
```

Comment:
This will break if we ever change this error message, but it should at least still throw whatever original error is occuring (and that's still informative, this one is just more specific).

Reply:
Added a comment in here since that's an actual helpful place to put this info.